data/ui/gametile.ui

```diff
@@ -54,6 +54,46 @@
       </object>
     </child>
   </object>
+  <object class="GtkPopover" id="menu_wine">
+    <property name="can_focus">False</property>
+    <child>
+      <object class="GtkBox">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="GtkModelButton" id="menu_win_winecfg">
+            <property name="visible">True</property>
+            <property name="can_focus">True</property>
+            <property name="receives_default">True</property>
+            <property name="text" translatable="yes" context="support" comments="Opens the official support webpage">Winecfg</property>
+            <property name="centered">True</property>
+            <signal name="clicked" handler="on_menu_button_winecfg_clicked" swapped="no"/>
+          </object>
+          <packing>
+            <property name="expand">False</property>
+            <property name="fill">True</property>
+            <property name="position">0</property>
+          </packing>
+        </child>
+        <child>
+          <object class="GtkModelButton" id="menu_win_regedit">
```

Comment:
Do we really need a regedit button? I think it's a bit much, it would probably be better without.